proposals/XXXX-compiler-version-directive.md

```diff
@@ -0,0 +1,79 @@
+# Compiler Version Directive
+
+* Proposal: [SE-XXXX](XXXX-compiler-version-directive.md)
+* Author: [David Hart](https://github.com/hartbit)
+* Review Manager: TBD
+* Implementation: [apple/swift#15977](https://github.com/apple/swift/pull/15977)
+* Status: TBD
+
+## Introduction
+
+This proposal deprecates the `_compiler_version` precompiler directive and replaces it with a `compiler` directive that is synctactically equivalent to the `swift` version check but checks against the latest version of the language the compiler comes with, regardless of which mode it's currently compiling in.
```

Comment:
S/precompiler/compiler

Reply:
I think the underscored directive can be removed from the proposal; it’s a hack whose underscore is meant to mean “don’t use this”